test/parallel/test-fs-realpath.js

```diff
@@ -225,7 +225,7 @@ function test_cyclic_link_protection(callback) {
   });
   assert.throws(() => {
     fs.realpathSync(entry);
-  }, common.expectsError({ code: 'ELOOP', type: Error }));
+  }, /^Error: ELOOP: too many symbolic links encountered, stat/);
```

Comment:
My apologies, I mean for errors that do not currently have a `code` property. These do.